index.html

```diff
@@ -14,48 +14,48 @@ <h1>Bloody Heck</h1>
     <div class="game-grid">
 
         <div class="game-card" id="game-card-1">
-            <img src="" alt="">
-            <h4></h4>
+            <img src="images/009-ninja.svg" alt="">
+            <h2>Sneaky</h2>
             <p></p>
         </div>
         <div class="game-card" id="game-card-2">
-            <img src="" alt="">
-            <h4></h4>
+            <img src="images/021-king.svg" alt="">
+            <h2>Nicholai</h2>
             <p></p>
         </div>
         <div class="game-card" id="game-card-3">
-            <img src="" alt="">
-            <h4></h4>
+            <img src="images/034-vampire.svg" alt="">
+            <h2>Sven</h2>
             <p></p>
         </div>
         <div class="game-card" id="game-card-4">
-            <img src="" alt="">
-            <h4></h4>
+            <img src="images/044-futuristic.svg" alt="">
+            <h2>Xia-borg</h2>
             <p></p>
         </div>
         <div class="game-card" id="game-card-5">
-            <img src="" alt="">
-            <h4></h4>
+            <img src="images/045-pirate.svg" alt="">
+            <h2>Polly</h2>
             <p></p>
         </div>
         <div class="game-card" id="game-card-6">
-            <img src="" alt="">
-            <h4></h4>
+            <img src="images/037-girl-1.svg" alt="">
+            <h2>Mary Sue</h2>
             <p></p>
         </div>
         <div class="game-card" id="game-card-7">
-            <img src="" alt="">
-            <h4></h4>
+            <img src="images/046-troglodyte.svg" alt="">
+            <h2>Winston</h2>
             <p></p>
         </div>
         <div class="game-card" id="game-card-8">
-            <img src="" alt="">
-            <h4></h4>
+            <img src="images//028-girl-3.svg" alt="">
+            <h2>Gertrude</h2>
             <p></p>
         </div>
         <div class="game-card" id="game-card-9">
-            <img src="" alt="">
-            <h4></h4>
+            <img src="images/029-rich.svg" alt="">
+            <h2>Mr. Millionaire</h2>
```

Comment:
Hmm, this guy looks like he has more money. Maybe let's call him Mr. Billionaire?

Reply:
Sounds good!